nextflow_schema.json

```diff
@@ -491,11 +491,15 @@
             "properties": {
                 "centrifuge_db": {
                     "type": "string",
+                    "format": "file-path",
+                    "pattern": ".*.tar.gz",
```

Comment:
Just checking - looks like supplying a directory as a centrifuge directory is supported, but would this kick back anything that's not a tar.gz?

Reply:
Ah crap yes, you're correct